tests/std/tests/P1208R6_source_location/env.lst

```diff
@@ -2,3 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 RUNALL_INCLUDE ..\usual_20_matrix.lst
+RUNALL_CROSSLIST
+*	PM_CL="" # Test default setting
+*	PM_CL="/D_USE_DETAILED_FUNCTION_NAME_IN_SOURCE_LOCATION=0" # Test escape hatch, see GH-4055

```

Comment:
Comments at the end of lines with configuration mean something to the internal test runner, let's not do this. I personally don't think we need the comment: we can search for `_USE_DETAILED_FUNCTION_NAME_IN_SOURCE_LOCATION` and figure out what's going on here very quickly. If disagree enough to restore the comments, please put them on their own lines above the configurations they describe.

Reply:
Guess it gas precedent here
https://github.com/microsoft/STL/blob/c7c5ca7d6adde7c1554b1e52c0b97cb2a7f7ca60/tests/std/tests/VSO_0000000_vector_algorithms/env.lst#L5-L7

Reply:
Thanks, @AlexGuteniev - I'll clean that up in a separate change.